frontend/src/Buildings/actionCreator.js

```diff
@@ -3,6 +3,9 @@ import { SERVER_URL } from 'react-native-dotenv';
 export const FETCH_BUILDINGS_REQUEST = 'fetchBuildingsRequest';
 export const FETCH_BUILDINGS_SUCCESS = 'fetchBuildingsSuccess';
 export const FETCH_BUILDINGS_FAILURE = 'fetchBuildingsFailure';
+export const ADD_BUILDING_REQUEST = 'addBuildingRequest';
```

Comment:
you haven't actually created the `ADD_BUILDING_REQUEST` and the `ADD_BUILDING_FAILURE` actions in this file, only the success

Reply:
I'll create another function addBuilding to replace the addBuildingSuccess when we'll connect to the backend in the future.